app/components/user_list/index.tsx

```diff
@@ -2,7 +2,7 @@
 // See LICENSE.txt for license information.
```

Comment:
I have the feeling this component is starting to do "too many things at the same time". We can leave it like this, but I guess we should revisit it in the future.

Reply:
Ticket created https://mattermost.atlassian.net/browse/MM-50035